lib/plugins/aws/deploy/index.js

```diff
@@ -32,16 +33,16 @@ class AwsDeploy {
       uploadArtifacts,
       updateStack,
       monitorStack,
-      configureStack
+      configureStack,
+      mergeIamTemplates
```

Comment:
Is this new lifecycle event AWS specific? Should we have a more general one in the main `deploy` plugin?